homeassistant/components/homeassistant/triggers/time.py

```diff
@@ -171,6 +179,19 @@ def update_entity_trigger(entity_id: str, new_state: State | None = None) -> Non
     to_track: list[str] = []
 
     for at_time in config[CONF_AT]:
+        # template.attach(hass, at_time)
+        # render = template.render_complex(at_time, variables, limited=True)
+        # at_time = _TIME_AT_SCHEMA(render)
+        if isinstance(at_time, template.Template):
+            template.attach(hass, at_time)
+            render = template.render_complex(at_time, variables, limited=True)
+            try:
+                at_time = _TIME_AT_SCHEMA(render)
```

Comment:
Could just use the same `_TIME_TRIGGER_SCHEMA` and don't need to make a new one without the check for a template?

Reply:
I tried that at first, but when it renders after the template, it should only allow time and the entity domains.  It was allowing integers, and other types because cv.template allows those to pass validation.  I wasn't sure how to get past this fault without separating the 2.  I'm not 100% familiar with vol.Schema, is it possible to make a schema and extend it while adding a message to it?  I couldn't find a way to do that.

Reply:
Looking at the code <https://github.com/alecthomas/voluptuous/blob/dcaaf3dd68be156253518a045feb1c4172dbd2d5/voluptuous/validators.py#L235>, msg is a property.  I could extend and change the message, however I still think I need the 2 validators.  Thoughts?

Reply:
I just looked into this more, it's not possible to extend the schema or use the same one.  I'd need to insert cv.template into the vol.Any object and it's not possible.  I'm going to leave this as-is.